### PR TITLE
Feature/loading animation

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

File: Assets/Script/Data/StoredStrings.cs [hunk 1]
```diff
@@ -192,4 +192,46 @@ public static class Doozy
...
+    {
+        /// <summary>
+        /// Prefix Tip
+        /// </summary>
+        public static string Tip => "Tip : ";
+
+        /// <summary>
+        /// Gravitational Orb Hint
+        /// </summary>
+        public static string GravitationalOrb => "The gravitational orb can suck your opponents towards the centre. Use that to direct them away from your path!";
```
Comment (mikeslgoh) [inline Assets/Script/Data/StoredStrings.cs]: Guess we should do some brainstorming for what to write for the tooltips - maybe for one of the meetings?

File: Assets/Script/UI/Loading/SpeedometerPointer.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    public class SpeedometerPointer : MonoBehaviour
+    {
+        [MinMaxSlider(-100, 100, true)]
+        [SerializeField]
+        private Vector2 m_pointerRange = new Vector2(-85, 85);
+        [SerializeField]
+        private float m_animationTime = 2f;
+        private Image m_pointer;
+        private Sequence mySequence = null;
+        private const float POINTER_OFFSET = 30f;
```
Comment (mikeslgoh) [inline Assets/Script/UI/Loading/SpeedometerPointer.cs]: what does this pointer offset do? And is the value set here from trial and error?

**Comment by mikeslgoh** [comment]
[Speedometer as loading bar](https://trello.com/c/vXDENwJK/391-speedometer-as-loading-bar)

**Comment by m4rtint** [comment]
[How to load loading screen between the different scenes](https://trello.com/c/0v6cdDvd/390-how-to-load-loading-screen-between-the-different-scenes)

File: Assets/Script/UI/Loading/SpeedometerPointer.cs (hunk 2 above)
Comment (m4rtint) [reply Assets/Script/UI/Loading/SpeedometerPointer.cs]: Offset is when the pointer go from 0 to x, like when you step on the gas

File: Assets/Script/UI/Loading/LoadingTooltip.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        {
+            m_elapsedTime += Time.deltaTime;
+            if (m_elapsedTime > m_hintTimeToLive)
+            {
+                SetTooltip(GetRandomText());
+                m_elapsedTime = 0;
+            }
+        }
+
+        private string GetRandomText()
```
Comment (DJSteins) [inline Assets/Script/UI/Loading/LoadingTooltip.cs]: Add documentation

Comment (m4rtint) [reply Assets/Script/UI/Loading/LoadingTooltip.cs]: Added documentation

**Comment by mikeslgoh** [comment]
@DJSteins can you help to approve since Martin has made the change? Thanks :)

File: Assets/Script/UI/Loading/SpeedometerPointer.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        /// It is called in UIView Show View Unity Event
+        /// </summary>
+        [Button]
+        public void StartPointerAnimation()
+        {
+            SetupPointer();
+            mySequence = DOTween.Sequence();
+            mySequence.PrependInterval(1);
+
+            for (int i = 1; i <= 6; i++)
```
Comment (wbeltran777) [inline Assets/Script/UI/Loading/SpeedometerPointer.cs]: What determines the choice of '6'?  Is it the number of 'keyframes' we're interpolating? Might be worth naming it.

File: Assets/Script/UI/Loading/SpeedometerPointer.cs [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+                mySequence.Append(tween);
+            }
+        }
+
+        private void Awake()
+        {
+            m_pointer = GetComponent<Image>();
+        }
+
+        private float Min()
```
Comment (wbeltran777) [inline Assets/Script/UI/Loading/SpeedometerPointer.cs]: What's the logic behind "Min" outputting 85 while "Max" outputs -85?

File: Assets/Script/UI/Loading/SpeedometerPointer.cs [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        [Button]
+        public void StartPointerAnimation()
+        {
+            SetupPointer();
+            mySequence = DOTween.Sequence();
+            mySequence.PrependInterval(1);
+
+            for (int i = 1; i <= 6; i++)
+            {
+                var destination = Mathf.Max(Min() - (i * POINTER_OFFSET), Max());
```
Comment (Gursaanj) [inline Assets/Script/UI/Loading/SpeedometerPointer.cs]: Remove Vars

File: Assets/Script/UI/Loading/SpeedometerPointer.cs (hunk 5 above)
Comment (m4rtint) [reply Assets/Script/UI/Loading/SpeedometerPointer.cs]: It's the rotation of the canvas. That's why i set it to Min/Max, it's just a black box that just works.

File: Assets/Script/UI/Loading/SpeedometerPointer.cs (hunk 6 above)
Comment (m4rtint) [reply Assets/Script/UI/Loading/SpeedometerPointer.cs]: will do

File: Assets/Script/UI/Loading/SpeedometerPointer.cs (hunk 4 above)
Comment (m4rtint) [reply Assets/Script/UI/Loading/SpeedometerPointer.cs]: Will name it.

**Pull request merged**